src/core/server/server.ts

```diff
@@ -174,21 +176,24 @@ export class Server {
       uiSettings: uiSettingsStart,
     });
 
-    const coreStart = {
+    this.coreStart = {
```

Comment:
Now that we need both `setup` and `start` contracts to register the core context in `registerCoreContext`, I need to a reference to the contract.